common/.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.2
```

Comment:
Is there a reason for this? We already have https://github.com/dependabot/dependabot-core/blob/efaca437c99258fdedf61ec7d6e498346aa59f44/.ruby-version

Reply:
Hello, 

Sorry no, I was testing some options in you read me in order to test it in my own custom usage with dependabot script, will remove it.